### DISCUSSION TITLE
new mod

### DISCUSSION THREAD

**Discussion by ReikoRei** [ppy/osu#24692] [Ideas] [answered]
Looking at the new map progress bar, I saw that it displays the easy and difficult moments of the map
and I thought it would be cool to add a mod that will slow down time on difficult parts of the map
so it will be possible to learn especially difficult sections of the map
of course you can play with half time or adaptive speed
but the last one just bad work
and with half time on easy parts of the map it becomes boring.

**Comment by peppy** [comment, marked as answer]
We have plans on adding a practice mod, which will allow you to hone in on sections you want to practice. I think this will work better than having a mod that does what you propose.

**Reply by Joehuu** [reply]
Original discussion at https://github.com/ppy/osu/discussions/13013.
